### ISSUE TITLE
[bug] `render_model` cannot show ∈ symbol in the saved file

### ISSUE THREAD

**Issue by dilaragokay** [pyro-ppl/pyro#3092]
### Issue Description
I created a simple model and rendered it the following way in Colab.
```python 
import pyro
import torch
from pyro.distributions import constraints, Uniform, LogNormal

def model(data):
    low = torch.tensor(0.)
    high = torch.tensor(5.)
    z = pyro.sample("difficulty", Uniform(low, high))

    sigma_grades = pyro.param("sigma_grades", torch.tensor(10.), constraint = constraints.positive)
    with pyro.plate("data", data.size(0)):
        pyro.sample("grades", LogNormal(90 - 10 * z, sigma_grades), obs = data)

data = torch.rand(70) * 100

pyro.render_model(model, model_args=(data,), filename='my_render.png',render_distributions=True, render_params=True)
```

I see in the notebook that the file is rendered the following way
<img width="680" alt="Screenshot 2022-05-23 at 12 16 25" src="https://user-images.githubusercontent.com/14012497/169798095-9452a4d6-ac59-4053-ab2c-12618644a189.png">

The above image is from a screenshot. When I download `my_render.png` and open the file, I see that the ∈ symbol is not shown correctly.
![my_render](https://user-images.githubusercontent.com/14012497/169798356-d572a3d1-673a-49f2-947c-9c999b8a8145.png)

Edit 1: I have just tried with a few more file extensions. Saving as an SVG works perfectly. Saving as a PDF does not work either. 
Edit 2: This might be a GraphViz issue actually: https://stackoverflow.com/questions/55456467/how-to-include-element-symbol-in-graphviz

### Environment
I am running this experiment on Colab. I installed Pyro in the first cell with `!pip3 install pyro-ppl`.
 - Python version: 3.7.13
 - PyTorch version: 1.11.0+cu113
 - Pyro version: 1.8.1

**Comment by fehiepsi** [comment]
I guess we can add a note in the docs that it is better to use `svg` format when `render_params=True`.

**Comment by fritzo** [comment]
we could just use a `:` symbol as in type theory and avoid the ∈ issue

**Comment by eb8680** [comment]
Should be addressed by #3097
